db/schema.rb

```diff
@@ -51,6 +51,7 @@
     t.text "body"
     t.bigint "user_id"
```

Comment:
what is  bigint here can we not use int only?

Reply:
integer holds values up to 2147483647, and bigint up to 9223372036854775807. Probably we can work with integer but id attributes are created by rails automatically and it seem like bigint is the default setting